app/views/categories/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render 'header' %>
+<main class="categories">
+  <div class="credit-info">
+    <p class="gift-text">Total Amount</p>
+    <p class="gift-amt"><%= Deal.where(author_id: current_user.id).sum(:amount) %>
+    </p>
+  </div>
+  <div class="categories-container">
+    <div class="category-scroll">
+      <% if @categories.present? %>
+        <% @categories.each do |category| %>
+          <%= link_to user_category_path(current_user, category) do %>
+            <div class="category-container">
+              <%= image_tag url_for(category.icon.variant(resize: "100x100")) if category.icon.attached? %>
+              <div class="cat-name-date">
+                <h2 class="cat-name"><%= category.name %></h2>
+                <p class="cat-date"><%= category.created_at %></p>
+              </div>
+              <span class="cat-total-amt"><%= category.deals.sum(:amount) %></span>
+            </div>
+          <% end %>
+        <% end %>
```

Comment:
- [ ] As suggested by the previous code reviewer, the uploaded icons of categories are broken, please fix it.

<img width="575" alt="image" src="https://github.com/ometman/budget-app/assets/52934128/9586c335-4485-4529-92fb-90dc161edcfe">

Reply:
Thanks. The bug is fixed. 😄